rllib/algorithms/algorithm_config.py

```diff
@@ -3905,18 +3913,11 @@ def validate_train_batch_size_vs_rollout_fragment_length(self) -> None:
         asking the user to set rollout_fragment_length to `auto` or to a matching
         value.
 
-        Also, only checks this if `train_batch_size` > 0 (DDPPO sets this
-        to -1 to auto-calculate the actual batch size later).
-
         Raises:
             ValueError: If there is a mismatch between user provided
                 `rollout_fragment_length` and `total_train_batch_size`.
         """
-        if (
-            self.rollout_fragment_length != "auto"
-            and not self.in_evaluation
-            and self.total_train_batch_size > 0
-        ):
+        if self.rollout_fragment_length != "auto" and not self.in_evaluation:
```

Comment:
Do we only support "auto" now?

Reply:
No, just the check for `train_batch_size` < 0 - now that we don't have the DDPPO algo anymore - seemed superfluous. So I took out that one check.